common.mk

```diff
@@ -15,6 +15,7 @@ ARCHES:=amd64 aarch64 armhf armel s390x ppc64le
 BUILDTIME=$(shell date -u -d "@$${SOURCE_DATE_EPOCH:-$$(date +%s)}" --rfc-3339 ns 2> /dev/null | sed -e 's/ /T/')
 CHOWN:=docker run --rm -v $(CURDIR):/v -w /v alpine chown
 DEFAULT_PRODUCT_LICENSE:=Community Engine
+COMPANY_NAME:=Docker, Inc.
```

Comment:
perhaps it should be empty by default (but we can set it in our internal pipeline)
that way, someone using the code from this repository doesn't set docker's name as packager